homework-submission/blog/src/Comments.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Update from './Update';
+
+class Comments extends React.Component {
+  state = {
+    update: false,
+    number: 0,
```

Comment:
it is a good improve to show the number of like, but it is not saved in the Database in the server, means when refresh the page the counter for likes will be ZERO 
that is not good for the user experience